### DISCUSSION TITLE
Delete APIs or CRDs in existing project

### DISCUSSION THREAD

**Discussion by hjiajing** [kubernetes-sigs/kubebuilder#2701] [Support] [answered]
I had created a kubebuilder project with several CRDs. Now that I want to delete some deprecated CRDs in the project. I wonder if there are some commands to delete them or if I have to edit the generated files which contain "DO NOT EDIT".

Thanks. 

According to PR#907. It seems that  I have to edit the files myself?

**Comment by camilamacedo86** [comment, marked as answer]
We do not have this feature, but it would be desirable. 
Currently, authors can do it manually by updating the apis/ and the Project file

WDYT about you raise an issue about it?
Also, would you like to collaborate with this project and provide a design solution?

**Comment by hjiajing** [comment]
Hi Camila. Thanks for your quick answer. It helps a lot. As for the issue, maybe I will raise an issue later. Anyway, thanks for the answer.

**Comment by camilamacedo86** [comment]
Closing since it is answered.
